legacy/ui/legacy/src/main/res/values-fa/strings.xml

```diff
@@ -950,4 +950,5 @@
     <string name="dialog_openkeychain_info_text">برنامه کاربردی OpenKeychain جهت فعال‌سازی رمزگذاری سرتاسری مورد نیاز است.</string>
     <string name="push_info_notification_explanation_text">در زمان استفاده از پیشرانی، <xliff:g id="app_name">%1$s</xliff:g> اتصال به کارساز را نگاه می‌دارد. اندروید نیازمند یک اعلان همیشگی جهت فعال ماندن برنامه در پس‌زمینه است. <xliff:g id="optional_extended_explanation">%2$s</xliff:g></string>
     <string name="about_app_authors_thunderbird">گروه همراه تاندربرد</string>
-</resources>
\ No newline at end of file
+    <string name="recipient_error_invalid_email">آدرس ایمیل نامعتبر است</string>
```

Comment:
Translations should only be updated via Weblate. Changing them in a pull request leads to the potential for merge conflicts in Weblate that are annoying to fix.